doc/sphinx-guides/source/installation/config.rst

```diff
@@ -3726,6 +3726,9 @@ please find all known feature flags below. Any of these flags can be activated u
     * - api-bearer-auth-use-builtin-user-on-id-match
       - Allows the use of a built-in user account when an identity match is found during API bearer authentication. This feature enables automatic association of an incoming IdP identity with an existing built-in user account, bypassing the need for additional user registration steps. This feature only works when the feature flag ``api-bearer-auth`` is also enabled. **Caution: Enabling this feature flag exposes the installation to potential user impersonation issues depending on the specifics of the IdP configured (For example, if it is configured such that an attacker can create a new account in the IdP, or configured social login account, matching a Dataverse built-in account).**
       - ``Off``
+    * - api-bearer-auth-use-shib-user-on-id-match
+      - Allows the use of a Shibboleth user account when an identity match is found during API bearer authentication. This feature enables automatic association of an incoming IdP identity with an existing Shibboleth user account, bypassing the need for additional user registration steps. This feature only works when the feature flag ``api-bearer-auth`` is also enabled. **Caution: Enabling this feature flag exposes the installation to potential user impersonation issues depending on the specifics of the IdP configured (For example, if it is configured such that an attacker can create a new account in the IdP, or configured social login account, matching a Dataverse built-in account).**
```

Comment:
Cut/paste error - I don't thing the caution fits, as minimally this doesn't involve matching the built-in account. It seems to me that the Shib case is stricter in that the idp would truly have to be malicious and lie about the shib provider name to be a threat (or not send expected shib attributes). (see other comment about whether the builtin user case can be made stricter). Otherwise, with a properly configured IdP, a malicious user can't create an account to take over someone else's - if I'm thinking correctly about it. If that's right, the warning might be reduced - Enabling this flag could result in impersonation risks if (and only if) used with a misconfigured IdP.